src/routes/raids/ViewRaid.tsx

```diff
@@ -19,62 +75,66 @@ const ViewRaid = () => {
       .collection('raid-stats')
       .doc(raidId)
       .get()
-      .then(snapshot => {
+      .then((snapshot) => {
         if (snapshot.exists) {
           setData(snapshot.data() as ViewRaidData)
           setState('success')
         } else {
           setState('not-found')
         }
       })
-      .catch(error => {
+      .catch((error) => {
         setError(error)
         setState('error')
       })
   }, [raidId])
 
   if (state === 'loading') {
-    return <LoadingSpinner />
+    return <p>Loading...</p>
   } else if (state === 'error') {
     return <p>{JSON.stringify(error)}</p>
   } else if (state === 'not-found') {
     return <p>Couldn`t find that Raid - did you fall into the wrong dungeon?</p>
   } else {
     return (
-      <main>
-        <header style={{ height: '7vh' }}>
-          <h1>{data?.title}</h1>
-          <p>
-            <Emoji as="🚀" aria-label="rocket" />{' '}
-            {Object.keys(data?.contributors ?? {}).length} Contributors
-          </p>
-          <p>
-            <Emoji as="💾" aria-label="floppy" /> {data?.commits}{' '}
-            {(data?.commits ?? 0) === 1 ? 'Commit' : 'Commits'}
-          </p>
-          <p>
-            <Emoji as="📃" aria-label="file" /> {data?.changedFiles} Changed
-            Files
-          </p>
-          <p>
-            <Emoji as="⚔️" aria-label="crossing-swords" /> +{data?.additions} -
-            {data?.deletions}
-          </p>
-          <p>
-            <Emoji as="🔥" aria-label="fire" />{' '}
-            {(data?.additions ?? 0) - (data?.deletions ?? 0)} Net Deletions
-          </p>
-        </header>
-        <ol style={{ overflowY: 'auto', height: '90vh' }}>
+      <StatsView>
+        <Header>
+          <div>
```

Comment:
Since we have a header, do we really need this `div`?

If we do, it is better to use `section` than `div` due to a11y.

Reply:
The div is just used for layout. I'll change it to section if that's better for a11y.

We will probably need to refactor this layout when we get the site header with navbar though..

Reply:
Let's just refactor it then I guess :+1: